src/components/Transcript/Transcript.js

```diff
@@ -33,14 +33,13 @@ const Transcript = ({ playerID, manifestUrl, transcripts = [] }) => {
     isMachineGen: false,
   });
   const [canvasIndex, _setCanvasIndex] = React.useState(0);
+  const [canvasIsEmpty, setCanvasIsEmpty] = React.useState(false);
```

Comment:
With the logic to explicitly update this being removed does this actually do anything? Is there background processing that happens that triggers this hook?

Reply:
No, it doesn't. Good catch!
This can be removed.